documentation/book/proc-instrumenting-kafka-streams-with-tracers.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// assembly-instrumenting-kafka-clients-tracers.adoc
+
+[id='proc-instrumenting-kafka-streams-with-tracers-{context}']
+= Instrumenting Kafka Streams for OpenTracing
+
+This section describes how to instrument Kafka Streams applications for distributed tracing.
+
+.Procedure
+
+Perform the following steps for each Kafka Streams application.
+
+. Add the `opentracing-kafka-streams` dependency to the pom.xml file for your Kafka Streams application:
++
+[source,xml,subs=attributes+]
+----
+<dependency>
+    <groupId>io.opentracing.contrib</groupId>
+    <artifactId>opentracing-kafka-streams</artifactId>
+    <version>VERSION</version>
```

Comment:
How will the user know which `VERSION` should this be? I guess we need to make this a variable?